.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Code owners file.
```

Comment:
We don't have Code owners file in any ApplicationInsights repo, why adding it here?

Reply:
This is not an ApplicationInsights repo.

Reply:
I know that, I want to know why you are adding this here?, what is the purpose of this file?, how can this be useful? ,etc. It feels this is irrelevant in a Microsoft owned repo but maybe I'm wrong

Reply:
The purpose of this file is so that any new issues will have the aliases in the CODEOWNERS file automatically tagged to review the PRs. It is not a REPO specific concept but a GITHUB concept.